handsontable/src/plugins/contextMenu/menu.js

```diff
@@ -33,11 +33,26 @@ import localHooks from '../../mixins/localHooks';
 
 const MIN_WIDTH = 215;
 
+/**
+ * @typedef MenuOptions
+ * @property {Menu} [parent] Default value: `null`.
```

Comment:
Please use JSDoc for describing default and use comment for introducing information what is represented by argument. It's general suggestion for almost all introduced JSDoc tags 🙂

```suggestion
 * @property {Menu} [parent=null] Instance of {@link Menu}.
```